examples/http_form.rs

```diff
@@ -0,0 +1,34 @@
+//! An example to show how to create a minimal health check server,
+//! using the [`HttpServer`] and [`Executor`] from Rama.
+//!
+//! [`HttpServer`]: crate::http::server::HttpServer
+//! [`Executor`]: crate::rt::Executor
+//!
+//! This example will create a server that listens on `127.0.0.1:8080`.
+//!
+//! # Run the example
+//!
+//! ```sh
+//! cargo run --example http_health_check
+//! ```
+//!
+//! # Expected output
+//!
+//! The server will start and listen on `:8080`. You can use `curl` to check if the server is running:
+//!
+//! ```sh
+//! curl -v http://127.0.0.1:8080
+//! ```
+//!
+//! You should see a response with `HTTP/1.1 200 OK` and an empty body.
+
+use rama::{http::server::HttpServer, rt::Executor, service::service_fn};
```

Comment:
Is this an example you are still planning to make? Because not sure if even needed. Either way for now it's just a copy paste of the health server example, so perhaps just delete this file? It's the last blocker for this PR To be able to be merged.

Reply:
Just a placeholder - not quite finished yet! :)

Reply:
Sure thing! not to worry. No pressure. I'll just leave you to it then. Take your time.
I suggest you just leave a comment here once you consider it final, and then we can take it from there to see if it can be merged in or not. Thanks for your dedication and work. Take care.